icgcget/cli.py

```diff
@@ -184,12 +112,30 @@ def download(ctx, repos, fileids, manifest, output,
 @click.option('--icgc-access', type=click.STRING)
 @click.option('--no-files', '-nf', is_flag=True, default=False, help="Do not show individual file information")
 @click.pass_context
-def status(ctx, repos, fileids, manifest, output,
+def status(ctx, repos, file_ids, manifest, output,
            cghub_access, cghub_path, ega_access, gdc_access, icgc_access,
            no_files):
     api_url = get_api_url(ctx.default_map)
-    gdc_ids, gnos_ids, repo_list = dispatcher.status_tables(repos, fileids, manifest, api_url, no_files)
-    dispatcher.access_checks(repo_list, cghub_access, cghub_path, ega_access, gdc_access, icgc_access, output, api_url)
+    dispatch = Dispatcher()
+    gdc_ids, gnos_ids, repo_list = dispatch.status_tables(repos, file_ids, manifest, api_url, no_files)
+    dispatch.access_checks(repo_list, cghub_access, cghub_path, ega_access, gdc_access, icgc_access, output, api_url)
+
+
+@cli.command()
+@click.option('--cghub-path', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.option('--ega-path', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.option('--ega-access', type=click.STRING)
+@click.option('--gdc-path', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.option('--icgc-path', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+def version(cghub_path, ega_access, ega_path, gdc_path, icgc_path):
+    dispatch = Dispatcher()
+    dispatch.version_checks(cghub_path, ega_access, ega_path, gdc_path, icgc_path)
```

Comment:
Is this really a version "check"? Isn't simply a listing? I think this terminology is confusing our conversations about it as well.